build.gradle.kts

```diff
@@ -20,6 +20,10 @@ buildscript {
     }
 }
 
+plugins {
+    alias(libs.plugins.jetpack.compose) apply false
+}
```

Comment:
duplication - compose plugin already added. to update version you should just change version in libs:
```
composeJetBrainsVersion = "1.1.1"
```